src/manifest.c

```diff
@@ -255,7 +256,11 @@ static int retrieve_manifest(int previous_version, int version, char *component,
 	/* Either we're not on mix or it failed, try curl-ing the file if link didn't work */
 	string_or_die(&filename, "%s/%i/Manifest.%s.tar", globals.state_dir, version, component);
 	string_or_die(&url, "%s/%i/Manifest.%s.tar", globals.content_url, version, component);
-
+	if (first_iteration) {
```

Comment:
Instead of printing this here, we can print it from the code where this is called, that way we can control where we want to print this. Consider this scenario:

In an operation we need to download a bunch of manifests, so we need to download the MoM first, when we download the MoM this is going to be set and we will print "Downloading manifest", but it was just the MoM so it was pretty quick. Then we may do some more stuff that prints other output in screen then we start downloading many manifests, at that point since we already retrieved a manifest this message won't be printed. And this is the moment where I'd like to see the message printed cause it may take a while downloading manifests.

Besides @otaviobp is going to complaint because of the use of a static variable 🤣

Reply:
Ya I removed ..will print this elsewhere